plans/example-browser-node/manifest.toml

```diff
@@ -0,0 +1,34 @@
+name = "example-browser-node"
+
+[defaults]
+builder = "docker:node"
+runner = "local:docker"
+
+[builders."docker:node"]
+enabled = true
+
+[builders."docker:generic"]
+enabled = true
+
+[builders."docker:generic".build_args]
+BROWSER_KIND = "firefox"
+BROWSER_DEBUG_PORT = "5321"
+HALT_BROWSER_ON_FINISH = "false"
```

Comment:
It should be possible to override these values at runtime right?

We should be able to build one image and override the browser variable when you start the container, which means we need the ability to pass custom env's to a testground plan.

Reply:
I agree, this is for now, as discussed on slack, a limitation of testground. As currently you cannot pass args to the `local:docker` runner, while you can do to the `docker:generic` builder. So once the first is possible as well we can get around that indeed.

Unless I am missing something of course.

Reply:
PR open: https://github.com/testground/testground/pull/1507

Reply:
Once the above PR is merged in, I'll merge master into this branch (again) and I'll switch over the logic I can move over :)

Reply:
At that point let's also to sit together so we can try to get the plan to work on the 3 browsers (success + failures) without requiring 3 different docker builds (not that I would know how to do that, as not sure how to pass args to the cfgs using build/run, not sure even possible?